src/components/get-started-with-TiDB-cloud.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+import { Link } from 'gatsby'
+import { Button } from '@seagreenio/react-bulma'
+
+import '../styles/components/get-started-with-TiDB-cloud.sass'
+
+const GetStartedWithTiDBCloud = React.memo(() => {
```

Comment:
Since this is not a common component, insert this section on `trust` page.

Reply:
this component is also used by pages/products/tidbcloud/index.js